src/main/resources/avro/referencemethods.avdl

```diff
@@ -284,6 +355,12 @@ record GetSequenceBasesRequest {
   union { null, long } end = null;
 
   /**
+  Specifies the maximum number of bases to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
```

Comment:
Sorry, I must have missed this before. When did we make a GET request that does paging? I thought the idea of the GETs was that they were simple requests that returned a single atomic object. 

In any case, this change is unrelated to the rest of the PR and should be put into it's own bugfix PR. It's not reasonable to expect people to re-review the entire PR each time there is an update.